app/sections/root/sections/tweets/tweets.component.ts

```diff
@@ -6,7 +10,73 @@ import {Component, Input} from 'angular2/core';
   styles: [
     <string> require('./tweets.styl')
   ],
+  directives: [...CORE_DIRECTIVES],
+  pipes: [ToDate]
 })
-export class TweetsComponent {
-  @Input() author:string;
+export class TweetsComponent implements OnInit {
+  @Input()
+  author:string;
+
+  private infFollow: Object= {
```

Comment:
I personally don't like to use type `Object` when smth has structure, could we use interface here to define type ? it depends on conventions, but anyway it is good to be consistent